x-pack/plugins/security_solution/public/timelines/store/helpers.ts

```diff
@@ -1456,7 +1456,7 @@ export const setInitializeTimelineSettings = ({
 }: InitializeTimelineParams): TimelineById => {
   const timeline = timelineById[id];
 
-  return !timeline?.initialized
+  const result = !timeline?.initialized
```

Comment:
why the change?

Reply:
some leftover from debugging. Removed it : 2863544e10efb1393126f2e42e723d2377a56408